content/docs/components/metadata.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "Metadata"
+description = "Tracking and managing metadata of machine learning workflows in Kubeflow"
+weight = 5
++++
+
+The goal of the [Metadata](https://github.com/kubeflow/metadata) project is to help Kubeflow users understand and manage their machine learning workflows by tracking and managing the metadata of workflows.
+
+## Installation
+
+The Metadata component is installed by default for Kubeflow versions >= 0.6.
+
+If you want to install the latest version of the Metadata component or install it as an application of your Kubernetes cluster, you can follow these steps.
```

Comment:
* `application of your Kubernetes cluster` or `application in your Kubernetes cluster` (`in` sounds more natural to me, but I don't know which is more correct)
* `steps.` --> `steps:` (introduce steps with colon)